modules/consentManagement.js

```diff
@@ -104,8 +104,8 @@ function lookupIabConsent(cmpSuccess, cmpError, hookConfig) {
         cmpSuccess(tcfData, hookConfig);
       } else if (tcfData.eventStatus === 'tcloaded' || tcfData.eventStatus === 'useractioncomplete') {
         cmpSuccess(tcfData, hookConfig);
-      } else if (tcfData.eventStatus === 'cmpuishown' && tcfData.tcString && tcfData.purposeOneTreatment === true) {
-        cmpSuccess(tcfData, hookConfig);
+      } else if (tcfData.eventStatus === 'cmpuishown') {
+        cmpSuccess(tcfData); // store consent, but don't continue to next hook
```

Comment:
@jsnellbaker - thoughts on this change?   Thinking that if the cmpTimeout fires before the user makes a selection we'd at least have whatever consent (past choice or the configured default?) was sent with the "cmpuishown" event.   This also came from the discussion in PR #5564 that you referenced.